### ISSUE TITLE
How to get NYUv2 normal GT of training set?

### ISSUE THREAD

**Issue by mt-cly** [ygjwd12345/TransDepth#22]
Hi, thanks for your work.
I notice that you refer to the https://github.com/MARSLab-UMN/TiltedImageSurfaceNormal in the issue https://github.com/ygjwd12345/TransDepth/issues/11 for  NYUv2 dataset. However, that project only provides the image & surface normal GT in terms of test set. I wonder how can I get the normal GT of NYUv2 training data so that dataloader can work well https://github.com/ygjwd12345/TransDepth/blob/main/surface_normal/dataset_loader/dataset_loader_nyud_2.py#L35 .

**Comment by ygjwd12345** [comment]
Hi, thanks for your attention. you can follow https://github.com/eccv2020anonymous/eccv2020-1886-release to get NYU's surface normal.

**Comment by mt-cly** [comment]
Thanks for your reply.
The link to download NYUv2 training GT provided at https://github.com/eccv2020anonymous/eccv2020-1886-release fail to access. Have you any other suggestions? Thanks.

**Comment by ygjwd12345** [comment]
you also can search how to use mat file of NYU to generate surface normal.
And another way is that you need to check geonet https://github.com/xjqi/GeoNet.

**Comment by ygjwd12345** [comment]
for me I don't save this data anymore

**Comment by ygjwd12345** [comment]
https://www.dropbox.com/s/dkvi87w8x3bqyg6/nyu-normal.zip?dl=0 
I am not sure this link which maybe my backup

**Comment by mt-cly** [comment]
thank you!
